src/compiler/checker.ts

```diff
@@ -42137,11 +42137,11 @@ namespace ts {
                 const { leftSpread, rightSpread, syntheticOrigin } = symbol as TransientSymbol;
                 return leftSpread ? [leftSpread, rightSpread!]
                     : syntheticOrigin ? [syntheticOrigin]
-                    : singleElementArray(tryGetAliasTarget(symbol));
+                    : singleElementArray(tryGetTarget(symbol));
             }
             return undefined;
         }
-        function tryGetAliasTarget(symbol: Symbol): Symbol | undefined {
+        function tryGetTarget(symbol: Symbol): Symbol | undefined {

```

Comment:
Seems like this function didn't need to be renamed?

Reply:
It was renamed because it handles `.target` rather than `.aliasTarget` lookups, for clarity.

Reply:
Ack, my comment was misguided because of GitHub's silent truncation of the diff. It looked like this function was handling `.aliasTarget` still. All good.